
# Send this whenever it is desired for SMARTS
# to restart the simulation with a new scenario:
string reset_with_scenario_path
